Sources/Extensions/CollectionViewManager+UICollectionViewDataSource.swift

```diff
@@ -15,6 +15,7 @@ extension CollectionViewManager: UICollectionViewDataSource {
     open func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cellItem = self.cellItem(for: indexPath)!
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: cellItem.reuseType.identifier, for: indexPath)
+        cellItem.cell = cell
```

Comment:
It's not a good idea to store cell inside cellItem. Cell item has `collectionView` and `indexPath` so you can just add computed property `cell` and call `UICollectionView` function `func cellForItem(at indexPath: IndexPath) -> UICollectionViewCell?'

Reply:
fixed